examples/window/transparent_window.rs

```diff
@@ -0,0 +1,32 @@
+/// An example of how to display a window in transparent mode
+/// [Documentation & Platform support.](https://github.com/bevyengine/bevy/blob/latest/crates/bevy_window/src/window.rs)
```

Comment:
Linking to the complete code file can be hard to follow. You could link to the exact line, but that could be a pain to keep track of changes.

Other possibility which I would prefer is to link to docs.rs. The field isn't already there for now, but it will be after the next release and its link should be  https://docs.rs/bevy/latest/bevy/prelude/struct.WindowDescriptor.html#structfield.transparent

Reply:
updated